modules/civicrmtheme/civicrmtheme.install

```diff
@@ -53,8 +53,40 @@ function civicrmtheme_install() {
  * Implementation of hook_uninstall().
  */
 function civicrmtheme_uninstall() {
-  // cleaning variables
-  variable_del('civicrmtheme_theme_admin');
-  variable_del('civicrmtheme_theme_public');
+
+}
+
+/**
+ * Implements hook_update_last_removed().
+ */
+function civicrmtheme_update_last_removed() {
+  return 7000;
```

Comment:
Is this correct? We should only have `hook_update_last_removed()` if the D7 version of this module included a `hook_update_7000()`. If the schema version if not 7000, then the update will refuse to run (AFAIK).